### PR TITLE
update schema

### PR DESCRIPTION
**Pull request by leonardomso**
# Summary

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅❌     |
| Android |    ✅❌     |

## Checklist

- [ ] I have tested this on a device and a simulator
- [ ] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [ ] I updated the typed files (TS and Flow)
- [ ] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

**Comment by leonardomso** [comment]
`yarn update-schema` was not working properly, so what I did was:

- Remove the `.webpack` folder and replace the `config`, `outputPath` and `outputFilename` to `webpackx.ts` file.
- Remove the `yarn w ./packages/server/scripts/updateSchema.ts` for `yarn workspace @workshop/server update-schema` just for get it working.

**Comment by leonardomso** [comment]
I want to work on another issues for now, for example:

- Added a few typings for inside the `web` packages, related to Relay. 
- Fix the `eslint --fix` which is broken due to some error related to `@testing-library/react/clear-after-each`.

File: solutions/02-useLazyLoadQuery/yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (sibelius) [inline solutions/02-useLazyLoadQuery/yarn.lock:1]: no need for this

**Comment by sibelius** [comment]
simpler solution here https://github.com/sibelius/relay-workshop/pull/80

please test

**Pull request closed without merging**